spec/person_spec.rb

```diff
@@ -0,0 +1,57 @@
+require_relative '../person'
+
+describe Person do
+  person = Person.new(23)
+  it 'Shoud create a person' do
+    expect(person.age).to eql 23
+  end
+
+  it 'Shoud check if person has no name' do
+    expect(person.name).to eql 'Unknown'
+  end
+
+  it 'Shoud check if person has no name' do
+    expect(person.name).not_to eql 'Something'
+  end
+
+  it 'Shoud Correct the name' do
+    person = Person.new(30, 'mhdez')
+    person.validate_name
+    expect(person.name).to eql 'Mhdez'
+  end
+  it 'Shoud check if person has parent permission' do
+    expect(person.parent_permission).to eq 'true'
+  end
+  it 'Shoud check if person has parent permission' do
+    expect(person.parent_permission).not_to eq 'false'
+  end
```

Comment:
- [ ] This test is the same like the one above, what the previous code reviewer meant when he said you also have to check for negative tests cases is that, you should consider for example the case where the parent_permission is false, like: `person2 = Person.new(15, name = 'Unknown', parent_permission = 'false')`
 `it 'Should not have parent permission' do`
    `expect(person2.parent_permission).to eq 'false'`
  `end`
Please kindly fix that.